Application/src/model/classDiagram/Relationships.java

```diff
@@ -1,19 +1,19 @@
 package model.classDiagram;
 
 /**
- * Stores fetchable data used for the GSON parser.
+ * Stores fetch-able data used for the GSON parser.
```

Comment:
no, it's definitely "fetchable"

Reply:
Intellij didn't like it and I don't like yellow errors.